site/content/docs/5.0/examples/global-headers/global-headers.css

```diff
@@ -0,0 +1,74 @@
+body {
+  padding-bottom: 20px;
+}
+
+.navbar {
+  margin-bottom: 20px;
+}
+
+.supra {
+  /* display: none; */
+  padding-top: 0;
+  padding-bottom: 0;
+  border-bottom: 1px solid $gray-900;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+  }
+
+  .navbar-nav {
+    margin: 0 -#{$navbar-supra-padding-x};
```

Comment:
This line (among other things) breaks the Netlify deployment. I know this is a still a WIP but fixing this could allow to see the results without having to deploy it locally 😃

Reply:
Yes, moreover I guess I shouldn't have done this in examples, as it will be in guidelines. I guess I can fix it by removing both html and css of global headers in /examples/